### PR TITLE
Feat / Add Tracks On The Fly 🦅🚀

### PR DESCRIPTION
**Pull request by johnxgaehner**
Going through each track in your collection is really time consuming so you can now add them on the fly from your playlist page. Just click "Add Tracks" and scroll through your collection to add/remove tracks - it's as easy as that. 🆒

https://user-images.githubusercontent.com/77127212/129881143-b02c95eb-eee8-4dd2-8971-b119cdff41fb.mov

### CODE REVIEW

**Comment by namirsab** [comment]
image

File: src/components/AddTrackOnTheFlyItem.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import { ReactComponent as IconSelectionEmpty } from "../assets/icon_selection_empty.svg";
+import { ReactComponent as IconSelectionFilled } from "../assets/icon_selection_filled.svg";
+import "./AddTrackOnTheFlyItem.css";
+
+export default function AddTrackOnTheFlyItem({
+  data,
```
Comment (namirsab) [inline src/components/AddTrackOnTheFlyItem.js:6]: In general, calling a prop `data` is not a good idea. (actually calling data to any variable is not really descriptive)

In any case, if you insist in calling it data, it wouldn't hurt to have `PropTypes` defined to know what this object should look like ;) 

https://dev.to/dcwither/stop-using-data-as-a-variable-name-3954

**Review by namirsab — APPROVED** [review]
Good job @johnxgaehner! 💪 
You are getting really fast and proficient already, that makes me really happy. Maybe its about time to think about refactoring some stuff (custom hooks, etc) as we talked yesterday? ;)

**Comment by patrikglanemann** [comment]
You are getting fancy here, you even added a video to show the feature😮 That is cool!
Looking forward to see what other features you implement to make it easier/more comfortable to use for the user!

**Pull request merged**